tasks/main.yml

```diff
@@ -13,15 +13,15 @@
     php_webserver_daemon: "{{ __php_webserver_daemon }}"
   when: php_webserver_daemon is not defined
 
-- name: Define php_conf_path.
+- name: Define php_conf_paths.
   set_fact:
-    php_conf_path: "{{ __php_conf_path }}"
-  when: php_conf_path is not defined
+    php_conf_paths: "{{ __php_conf_paths }}"
+  when: php_conf_paths is not defined
```

Comment:
This will probably need to be wrapped in {{}} so no DEPRECATION warning pops up.